.travis.yml

```diff
@@ -1,21 +1,45 @@
 os: linux
 dist: xenial
 language: python
+cache: pip
 python: 3.7
 sudo: true
-
 services:
-  - docker
-
-matrix:
+- docker
+env:
+  global:
+  - secure: p2k4l8RmT5JX81rmesKh8MQoJiO5LowU+vfQxeuzvilFAByuf7FaShaV0TMCM2rt3pUa7YnbklMfHI8KA3y1D2D0ShYXIO7YKmpQNYRmlV7hbfnWE8ySeVtuSopwQObivZjfeSbtJfpd4yJHLTxOoicja9LaMRU/UvUgZX6qeAE6ITzZZBe+cT7b4ekD2CGPrsdpCdU3IVxfZjup9jSVzwZOBgZ3PJmTtoI5a4V9jz5SCohUXcV5vq63SkHaKleHEaYpZjdddVrQKtXdfjYB/DDqAgOyYMqY6lAVk2fdRHE+U3oobcuoTa7TP7Gdg/zRvgq9rHCad270duGgLkmqwrCPaCB+QMyEQ3P4t+PYp3mUg4H8xwYPBiW1T0aY3S3ZqnTF1hPbs1TK7Fy8I7RAoroGZyTRlICbeghUul6OzDHXyu+fbe05Jw/qIGbe7ko5vc1bfNotRmbI16KlLhCdXeWWRtv8UWpVIIGVpMLmeOW3vvp1/kz8LUhKg+eXhxOxQpybh5M/z8+pr8L1ZZzvhCu7cVycmKMU1toh5nvHQopEwJ2l8KMe/h3X9DNL0r/OQNke0KhPbhGhiXdZIKxPdH08kyZx/TldA6IK4k/1Ze+CssUSCI3MqkjqkQeqlFB3ciwo4xX1Oiycd1v63+fHyVQpIexrdS3MSeyQgTRc59M=
```

Comment:
These were generated as follows: 
```
$ gem install travis
$ travis encrypt DOCKER_USER=<my_dockerhub_username> --add
$ travis encrypt DOCKER_PASS=<my_dockerhub_password> --add
```

Reply:
OK This seems like a comment that needs to be added to the documentation/workflow somewher